src/lando/ui/jinja2/stack/partials/landing-preview.html

```diff
@@ -3,10 +3,11 @@ <h3 class="StackPage-landingPreview-sectionLabel">Landing is Blocked</h3>
   <div class="StackPage-landingPreview-section StackPage-landingPreview-blocker">
     Reason for blockage is unknown
   </div>
-{% elif dryrun['blocker'] %}
+{% elif dryrun['blocker'] or dryrun['blockers'] %}
+  {% set blocker = dryrun['blockers'][0] if 'blockers' in dryrun else dryrun['blocker'] %}
   <h3 class="StackPage-landingPreview-sectionLabel">Landing is Blocked</h3>
   <div class="StackPage-landingPreview-section StackPage-landingPreview-blocker">
-    {{ dryrun['blocker']|escape_html|linkify_faq|safe }}
+    {{ blocker|escape_html|linkify_faq|safe }}
```

Comment:
It might be better to do add `blocker` to `blockers` and loop over `blockers` rather than showing `blockers[0]` only?